frontend/src/scss/PrimeStyles.scss

```diff
@@ -77,8 +77,8 @@
   background: none !important;
   border: none;
   padding: 0 !important;
-  color: #069;
-  text-decoration: underline;
+  color: #069 !important;
+  text-decoration: underline !important;
```

Comment:
I understand the color since it will use a different one for a visited link. What was overriding the underline?

Looking at https://designsystem.digital.gov/components/button/ we are supposed to be able to avoid this if we use their unstyled button variant. Then instead of using our `Anchor` we could just use `<Button unstyled onClick={...} `. Our current `Button` component doesn't have `unstyled` built in but we could add it.

Reply:
Reg underline override:
![image](https://user-images.githubusercontent.com/74203452/101414753-8eb14280-389b-11eb-9c50-987b01a39be5.png)

![image](https://user-images.githubusercontent.com/74203452/101415000-fa93ab00-389b-11eb-8311-f4ac09f4b458.png)


Yeah, I wasn't sure what was best to use. I considered using `<Anchor>` to be the clearest but that component doesn't allow for a style override. 

The TopNav really does not want links to look like links (since each have `<NavLink>` tab is a link).

I could undo this `!important` and modify `<Anchor>` to allow for a classname prop.